andela/travel_tool_front
# #161176885 Prevent image re-upload on edit

#### What does this PR do?
Fixes unnecessary image re-upload on edit while the user does not change the previous images

#### Description of Task to be completed?
Make sure there is no image re-upload if the user doesn't edit the image

#### How should this be manually tested?
- Clone the repo `https://github.com/andela/travel_tool_front.git`
- Checkout to the branch:
```bg-image-reupload-edit-161176885```
- Install dependancies:
```yarn install```
- Start the application:
```yarn start```

- Login to the application
- Navigate to the manage page and click on a guest house center to view it's details
- Click on the edit link make some changes in the input field except for the image.
 - Right click on the browser and click on Inspect and choose the network tab to view the expected behaviour when saving the changes.
- Save the changes.
- The image doesn't reupload anymore if you never edited it.

#### Any background context you want to provide?
You should have the user role of a Travel Administrator or Super Administrator to be able to see the `manage` page. To do that:
- Download `Postico` and connect to your travela database with relevant details: Db name, password, port, user
- Click `userRoles` table and set the `roleId` to that of a Travel Administrator(29187) or Super Administrator(10948) 


#### What are the relevant pivotal tracker stories?
[161176885](https://www.pivotaltracker.com/story/show/161176885)
